standards/git-commands.md

```diff
@@ -0,0 +1,34 @@
+# Git Commands
+Git is a distributed version-control system for tracking changes in source code during software development. It is designed for coordinating work among programmers, but it can be used to track changes in any set of files. Its goals include speed, data integrity, and support for distributed, non-linear workflows.
+
+At Shift3, we use git for a majority of our software projects.
+
+## Shift3 git specific resources
```

Comment:
I think we should be consistent about how we capitalize words in titles. Do we want to capitalize all words, as in `Git Commands` above and `Git Stash` below? Not high priority just something I noticed.

Reply:
Nice catch, this should fix the titles: 46a852e9612e94dc7955cb2810dccd0e0c1474c1